### DISCUSSION TITLE
Trim command not working

### DISCUSSION THREAD

**Discussion by nmlizard** [ImageMagick/ImageMagick6#125] [Q&A]
Hi, 

I am having issues with trying to Trim the white border from a batch of scanned images of old photos. I thought perhaps it was because the photos are flush on the north side, so I tried adding a border before trimming, but it didn't work. I am definitely a newbie, so I appreciate help!

Here's the command I have been using: 

`magick mogrify -path {output path here} -bordercolor White -border 10 -fuzz 5% -trim +repage *.jpg`

The images come from an SD card (from a little FlipPal mobile scanner I use only for old family photos). Unfortunately, I have been unsuccessful batch-trimming or auto-trimminb these files on any program, and there are hundreds of them. Any suggestions much appreciated.

Thank you!

**Comment by urban-warrior** [comment]
Increase your fuzz from 5% to 15% to 40%.  Does that work?  Old family photos may include a lot of distortions on the edges.  If so, you might not be able to auto-trim them.  Instead you may need to manually crop each one.  Or you may need deep learning to auto-crop.  ImageMagick does not offer deep learning.
